src/components/wrapper/wrapper.test.js

```diff
@@ -32,11 +32,16 @@ describe('Wrapper', () => {
     expect(container.length).toBe(0);
   });
 
+  it('does not display the settings modal when displayGlobalToolbar is false', () => {
+    const wrapper = setup.mount(Wrapper, mockPropsNoGlobalToolbar);
+    const container = wrapper.find('.pipeline-settings-modal');
+    expect(container.length).toBe(0);
+  });
+
```

Comment:
In looking at this test, this test doesn't serve any purpose / isn't exactly testing anything as the settings modal won't be displayed even if `displayGlobalToolbar` is true ( `.pipeline-settings-modal` is only triggered when the modal is triggered and not on the existence of the globalToolbar). 

This could be deleted and ideally, we should set up a test to mock the navigation to one of the routes (such as `/experiment-tracking`) and ensuring that the app doesn't get routed when `displayGlobalToolbar` is false. ( the test condition would be something like 'does not perform any routing when displayGlobalToolbar is false').